### PR TITLE
Show recommended subscriptions

### PR DESCRIPTION
**Pull request by seanyesmunt**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Breaking changes (bugfix or feature that introduces breaking changes)
- [x] Code style update (formatting)
- [x] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: #1825

## What is the current behavior?
We don't recommend any channels for subscriptions

## What is the new behavior?
Channels are recommended on your first run, if you clear out all of you unread subscriptions, and if you unsubscribe from all of the channels you originally subscribed too.

## Additional changes
- Cleaned up the `util/` folder. Now all files use the same naming convenstions `file-name.js`
- Split the subscriptions page into sub components, it was starting to get very big.

### CODE REVIEW

File: src/renderer/component/subscribeButton/view.jsx [hunk 1]
```diff
@@ -17,23 +17,31 @@ type Props = {
   doChannelSubscribe: ({ channelName: string, uri: string }) => void,
   doChannelUnsubscribe: SubscribtionArgs => void,
   doNotify: ({ id: string }) => void,
+  firstRunCompleted: boolean,
 };
 
 export default (props: Props) => {
   const {
-    channelName,
```
Comment (seanyesmunt) [inline src/renderer/component/subscribeButton/view.jsx]: Before we were passing the uri and the channel name in, but we can just get the channel name from the uri

File: src/renderer/component/subscribeButton/view.jsx [hunk 2]
```diff
@@ -42,14 +49,19 @@ export default (props: Props) => {
       onClick={e => {
         e.stopPropagation();
```
Comment (skhameneh) [inline src/renderer/component/subscribeButton/view.jsx]: I would suggest moving this logic out of the inline markup

File: src/renderer/store.js [hunk 3]
```diff
@@ -116,14 +115,7 @@ const persistOptions = {
...
   // read the data
-  transforms: [
-    subscriptionsFilter,
-    walletFilter,
-    contentFilter,
-    fileInfoFilter,
-    appFilter,
-    compressor,
-  ],
+  transforms: [walletFilter, contentFilter, fileInfoFilter, appFilter, compressor],
```
Comment (skhameneh) [inline src/renderer/store.js]: Why the change in formatting?
*Mostly curious

Comment (seanyesmunt) [reply src/renderer/store.js]: I removed the `subscriptionsFilter` so prettier decided it could all be on one line
